src/services.tsx

```diff
@@ -31,4 +33,25 @@ const auth = async (user: User, url: string): Promise<AuthPops | undefined> => {
 	return result;
 };
 
-export { auth };
+const getCoursesList = async (): Promise<CourseResponse> => {
+	const response: Response = await fetch('http://localhost:4000/courses/all');
+	const coursesList: Promise<CourseResponse> = response.json();
+
+	return coursesList;
+};
+
+const getCourseInfo = async (id: string): Promise<CourseInfoResponse> => {
+	const response: Response = await fetch(`http://localhost:4000/courses/${id}`);
+	const courseInfo: Promise<CourseInfoResponse> = response.json();
+
+	return courseInfo;
+};
+
+const getAuthorsList = async (): Promise<AuthorResponse> => {
+	const response: Response = await fetch('http://localhost:4000/authors/all');
+	const authorsList: Promise<AuthorResponse> = response.json();
+
+	return authorsList;
+};
```

Comment:
can't see any error handling here.

if you have it at places you are using these functions then it is better to move them here